packages/embed/src/utils/create-custom-launch-options.ts

```diff
@@ -69,3 +81,12 @@ export const handleCustomOpen = (open: () => void, openType: string, value?: num
     // do not open automatically
   }
 }
+
+const getPreventOpenOnCloseCookieValue = (formId: string) => {
+  const match = document.cookie.match(new RegExp(`(^| )tf-${formId}-closed=([^;]+)`))
+  return (match && match[2]) || undefined
```

Comment:
Since we know the value, I think we could check like this and get a boolean return value:

```suggestion
  return document.cookie.includes(`tf-${formId}-closed=true;`)
```

I would disregard the fact that there might be cookies with similar name and different prefix.